deptry/cli.py

```diff
@@ -212,6 +220,17 @@ def deptry(
         logging.warning("Missing argument ROOT. E.g. `deptry .`")
         sys.exit(1)
 
+    virtual_env = os.environ.get("VIRTUAL_ENV")
```

Comment:
There are quite some lines of code here, at a lower abstraction level than the other function calls in `cli.py`. Maybe we can create a class `VirtualEnvironmentDetector` (or a more approriate name if you have another proposal), that is called here. and move this code in a e.g. `detect()` method.

Reply:
Refactored to expose single entrypoint!